package.json

```diff
@@ -32,7 +34,8 @@
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "test": "yarn test:jest",
-    "pretest": "yarn lint"
+    "pretest": "yarn lint",
+    "declaration:build": "tsc --allowJs --declaration --emitDeclarationOnly sdk.js"
```

Comment:
Please add this command to the `lintstaged` config so that it gets generated whenever there's a change to the original source file.

Reply:
I've updated the linststaged config, if you want I can add it to the release command